docker-compose.yml

```
@@ -1,4 +1,13 @@
 services:
+  celestia-init:
+    image: ghcr.io/celestiaorg/celestia-app-standalone:02ff9af
+    container_name: celestia-init
+    entrypoint: /scripts/init.sh
+    volumes:
+      - ./scripts/init-celestia.sh:/scripts/init.sh:ro
+      - celestia-app:/home/celestia/.celestia-appd
```

[not blocking] ditto

```suggestion
      - celestia-app:/home/celestia/.celestia-app
```